content/docs/start/index.md

```diff
@@ -67,6 +67,11 @@ by one in the next few pages:
   data artifacts are built, and provide an efficient way to reproduce them.
   Think "Makefiles for data and ML projects" done right.
 
-- [**Experiments**](/doc/start/experiments) attach parameters, metrics, plots.
-  You can capture and navigate experiments without leaving Git. Think "Git for
-  machine learning".
+- [**Metrics, parameters, and plots**](/doc/start/metrics-parameters-plots)
+  explains how to attach metrics, parameters, and plots. You can capture,
+  navigate, and evaluate ML projects without leaving Git. Think "Git for machine
+  learning".
+
+- [**Experiments**](/doc/start/experiments) enable exploration, iteration, and
+  comparison across many ML experiments. Track your experiments with all of the
+  reproducibility and power of DVC.
```

Comment:
> Track your experiments with all of the reproducibility and power of DVC.

Sounds nice but not sure what it means. How about mentioning a few features/solutions? E.g. automatic versioning, log-like runtime checkpoints, rolling-back/resuming from different points.

Reply:
Let me know your thoughts on my revisions here

Reply:
> > How about mentioning a few features/solutions? E.g. automatic versioning, log-like runtime checkpoints
>
> Let me know your thoughts on my revisions

Yeah great, thanks for the updates. Well, it may be too detailed now. Up to you 🙂 Resolving here